scripts/test.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Exit immediately if a command exits with a non-zero status.
+set -e
+
+# prepare apps
+./scripts/build-all-apps.sh
```

Comment:
This assumes we're in the root to execute this, yeah?

let's `cd "$(dirname $0)"`, so we can then call `./build-all-apps`